File: python/docs_code_snippets/traces-list.py
from opperai import Opper

opper = Opper()

# --- docs ---
traces = opper.traces.list(limit=5)
for t in traces.data:
    print(f"{t.id} - {t.name or '(unnamed)'} ({t.span_count} spans)")
# --- /docs ---

File: typescript/docs_code_snippets/traces-list.ts
import { Opper } from "opperai";

const opper = new Opper();

const traces = await opper.traces.listTraces({ limit: 5 });
for (const t of traces.data) {
  console.log(`${t.id} - ${t.name ?? "(unnamed)"} (${t.span_count} spans)`);
}
